Modules/Setting/Services/CatalogueGetService.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Modules\Setting\Services;
+
+use Config;
+use ThrowException;
+
+/**
+ * Class CatalogueGetService
+ * @package Modules\Catalogue\Services
+ */
+class CatalogueGetService {
+    /**
+     * @var array $catalogue
+     */
+    private $catalogue;
+
+    /**
+     * CatalogueGetService constructor.
+     */
+    public function __construct() {
+        $this->catalogue = [];
+    }
+    /**
+     * @param array $data
+     * @return array|null
+     */
+    public function list(array $data) : ?array {
+        foreach ($data['filters'] as $var)
+        {
+            $this->catalogue[$var] = Config::get("catalogue.$var");
+        }
+
+        return $this->catalogue;
```

Comment:
no declares $this->catalogue como propiedad
el 99% de los casos una clase debe de ser "stateless" 
eso significa que no tiene otros atributos mas que los que inyectas.
En este caso solo has una variable $setting antes del foreach en la linea 29 y retorna esa variable.

Reply:
entendido.